packages/ckeditor5-emoji/src/emojimention.ts

```diff
@@ -81,9 +81,21 @@ export default class EmojiMention extends Plugin {
 		this._emojiDropdownLimit = editor.config.get( 'emoji.dropdownLimit' )!;
 		this._skinTone = editor.config.get( 'emoji.skinTone' )!;
 
-		const mentionFeedsConfigs = editor.config.get( 'mention.feeds' )! as Array<MentionFeed>;
+		this._setupMentionConfiguration( editor );
+	}
+
+	/**
+	 * Initializes the configuration for emojis in the mention feature.
+	 * If the marker used by emoji mention is already registered, it displays a warning.
+	 * If emoji mention configuration is detected, it does not register it for a second time.
+	 */
+	private _setupMentionConfiguration( editor: Editor ) {
+		const mentionFeedsConfigs = editor.config.get( 'mention.feeds' )! as Array<EmojiMentionFeed>;
 		const mergeFieldsPrefix = editor.config.get( 'mergeFields.prefix' )! as string;
-		const markerAlreadyUsed = mentionFeedsConfigs.some( config => config.marker === EMOJI_MENTION_MARKER );
+		const isEmojiMarkerAlreadyAdded = mentionFeedsConfigs.find( config => config._isEmojiMarker );
+		const markerAlreadyUsed = mentionFeedsConfigs
+			.filter( config => !config._isEmojiMarker )
```

Comment:
Commenting out this line `.filter( config => !config._isEmojiMarker )` causes that all tests still pass correctly, so probably one more unit test is needed.

Reply:
Let's create two editors, but the second one uses a configuration ('mention.feeds') from the first one. Both must load the `EmojiMention` plugin. This way it should verify the fix and confirm that we fix the issue.

Reply:
Added in [61774d6](https://github.com/ckeditor/ckeditor5/pull/17826/commits/61774d6abb6decd570dea22415700995f91c5266)